client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -285,7 +284,9 @@ const RecurringPaymentsPlanAddEditModal = ( {
 				</FormFieldset>
 				<FormFieldset>
 					<FormToggle onChange={ handleMultiplePerUser } checked={ editedMultiplePerUser }>
-						{ translate( 'Allow the same customer to sign up multiple times to the same plan.' ) }
+						{ translate(
+							'Allow the same customer to purchase or sign up to this plan multiple times.'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.